tutor/src/screens/assignments/details.js

```diff
@@ -44,10 +44,16 @@ const Details = ({ ux }) => {
             <HintText>(Apply a pre-set submission and grading policy template)</HintText>
           </Label>
           <div>
-            <Select name="grading_template">
+            <Select
+              name="grading_template"
+              onChange={e => {
+                ux.selectedGradingTemplateId = e.target.value;
+                ux.form.setFieldValue('grading_template', e.target.value);
+              }}
```

Comment:
@nathanstitt Wondering if there's a better way than overriding to get the selected value. I originally tried checking `form.values.grading_template` directly in a computed, but it doesn't get updated until the step changes.

Reply:
hm, why do we need to set the `ux.selectedGradingTemplateId` ?  If we change the name here to `grading_template_id`, this should set `ux.fom.values.grading_template_id` and UX can then use as needed.

I think we'd have to set add a `@field grading_template_id` on the plan, but we'll need to do that anyway so it saves